packages/expo-splash-screen/ios/EXSplashScreen/EXSplashScreenController.m

```diff
@@ -17,15 +17,15 @@ @interface EXSplashScreenController ()
 
 @implementation EXSplashScreenController
 
-- (instancetype)initWithViewController:(UIViewController *)viewController
-              splashScreenViewProvider:(id<EXSplashScreenViewProvider>)splashScreenViewProvider
+- (instancetype)initWithRootView:(UIView *)rootView splashScreenView:(nonnull UIView *)splashScreenView
 {
   if (self = [super init]) {
-    _viewController = viewController;
+    _rootView = rootView;
     _autoHideEnabled = YES;
     _splashScreenShown = NO;
     _appContentAppeared = NO;
-    _splashScreenView = [splashScreenViewProvider createSplashScreenView];
+    _splashScreenView = splashScreenView;
+    _splashScreenView.userInteractionEnabled = YES;
```

Comment:
for normal usage, the splash screen view does not need to be interactive.
how about moving this to `EXManagedAppSplashScreenController`?